testdata/p4_16_errors_outputs/issue3057-1.p4-stderr

```diff
@@ -0,0 +1,6 @@
+issue3057-1.p4(11): [--Werror=type-error] error: ==: cannot compare initializers with unknown types
```

Comment:
this message isn't very good. I know you didn't change this, but perhaps you can fix it.
The message should say "cannot compare structure-valued expressions with unknown types".

Reply:
I expect that you will need to change these reference files now that the error message has changed.